src/components/MediaGallery/MediaGallery.tsx

```diff
@@ -38,6 +39,14 @@ const MediaGallery = ({ resources: initialResources, tag }: MediaGalleryProps) =
   const [selected, setSelected] = useState<Array<string>>([]);
   const [creation, setCreation] = useState<Creation>();
 
+  useKeydown({
+    key: "Escape",
+    handler: (_event: KeyboardEvent) => {
+      setSelected([]);
+    },
+    element: document.body
```

Comment:
i think this is what's making the build fail. i suspect because it's in the render sequence and if this is trying to render on the server, document won't be available. i'm not sure if just a simply optional chain will fix it? `document?.body`? or perhaps it needs to be a ref?

<img width="1159" alt="image" src="https://github.com/cloudinary-community/photobox/assets/1045274/d32e06ce-7ab0-414c-9030-bcfdbc463e94">